commands/trivia_command.py

```diff
@@ -1,15 +1,88 @@
 import requests
 
+from constants import PARSE_MODE_KEY, EDIT_MESSAGE_ENDPOINT
+from config import BASE_URL
 from .botcommand import BotCommand
 from boterror import BotError
 from botquery import BotQuery
+from trivia import Trivia, TRIVIA_BASE_ERROR_MESSAGE
 
 class TriviaCommand(BotCommand):
+    def __init__(self, query, command_arguments=None):
+        super().__init__(query, command_arguments)
+        self.trivia = None
+
     def execute(self):
-        pass
+        if self.query.has_callback_query:
+            self.handle_callback_query()
+            return
```

Comment:
This looks like it should just be a separate method. How about having two methods: `execute` and `execute_callback_query`. Main will automatically call `execute_callback_query` on any command that supports callback_queries, when the parser determines that the query is a callback_query. I don't like funnelling everything into execute, but then doing an early return and calling another method.